### ISSUE TITLE
Incompatible Bootstrap - which viewset to use: views-cities2 or views-cities3?

### ISSUE THREAD

**Issue by senguttuvang** [lightspeedretail/webstore#676]
I assumed views-cities3 to be the logical upgrade to views-cities2, but it doesn't seem to be the case. Strangely, views-cities2 is based on Bootstrap3, while the views-cities3 is based on Bootstrap2. The default install automatically chooses views-cities3. Why a latest viewset should use older Bootstrap?

Also, the jquery comes with Yii framework included in this package is 1.8.x while Bootstrap3 demands 1.9.1? Shall I configure the clientScript component to adjust the latest jQuery library?

In short, what is the suggested method to use Bootstrap3?

**Comment by ottaz** [comment]
The folders prefixed with `views` are not actually upgrades of each other. They are used to govern different user experiences through **themes**.  Our currently supported themes are Brooklyn and Brooklyn 2014 both of which were built using the now deprecated version 2 of Bootstrap. Brooklyn uses the view-cities folder and Brooklyn-2014 uses the view-cities3 folder. We had a few other themes that used view-cities2 which utilitizes bootstrap3 but have since made those unavailable.

You should be safe to just register the library directly from the CDN.
Something like,
```php
Yii::app()->getClientScript()->registerScriptFile('https://code.jquery.com/jquery-1.9.1.min.js');
```
should theoretically work.
